CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
+## 0.12.0
```

Comment:
Patch version?

Reply:
Add an empty line to match the other log entries.

Reply:
I thought I added a functionality in a backwards compatible way, that's way I updated the minor.

Reply:
Some time ago we agreed to bum minor in case of backward-incompatible changes and patch otherwise (in case we're on dev mode, major version == 0)

Reply:
It is not the first time I found a discrepancy related to versioning. May I ask to write the agreement somewhere? That [page](https://github.com/yola/wiki/wiki/Library-Best-Practices) looks like a good candidate.

Reply:
Also, the current repository is a public library, so I believe we cannot relate to internal conventions. People will expect semantic versioning without any added flavor.

Reply:
Sure, semver says that during dev mode everything can change.  We follow semver in our convention.

Reply:
Updated https://github.com/yola/wiki/wiki/Library-Best-Practices